addons/account/models/account_payment_method.py

```diff
@@ -110,7 +110,7 @@ class AccountPaymentMethodLine(models.Model):
         domain="[('deprecated', '=', False), "
                 "('company_id', '=', company_id), "
                 "('account_type', 'not in', ('asset_receivable', 'liability_payable')), "
```

Comment:
Let's remove this line in master during the forward port, it doesn't make any sense